### DISCUSSION TITLE
How does WindowsKey.VolumeMute work?

### DISCUSSION THREAD

**Discussion by JAng-A3** [Voltstro-Studios/UnityWebBrowser#354] [Q&A]
As part of a project to have multiple web browsers present in a scene, there is a requirement to mute audio on command. I thought it would be possible by using the SendKeyboardControls along with the  WindowsKey.VolumeMute key, but after several attempts it does not seem to work.

Could you please enlighten me as to how to properly use the VolumeMute key?

Example code:

```WindowsKey[] keyDown = new WindowsKey[1];
keyDown[0] = WindowsKey.VolumeMute;
WindowsKey[] keyUp = new WindowsKey[1];
keyUp[0] = WindowsKey.None;
char[] character = Array.Empty<char>();
webBrowserClientInputHandler.browserClient.SendKeyboardControls(keyDown, keyUp, character);

**Comment by Voltstro** [comment]
Don't know CEF handles `WindowsKey.VolumeMute`, but #357 is done now.
